doc/10-icinga-template-library.md

```diff
@@ -1725,6 +1725,21 @@ Name             | Description
 users\_win\_warn | **Optional**. The warning threshold.
 users\_win\_crit | **Optional**. The critical threshold.
 
+### file-age-windows <a id="windows-plugins-file-age-windows"></a>
+
+Check command object for `check_file_age.cmd` command file and `check_file_age.cmd.ps1` plugin.
+
+Custom variables:
+
+Name             | Description
+:----------------|:------------
+file             | **Required**. File name and location
```

Comment:
`file` is nor correct here, this must have the prefix as set in the CheckCommand definition. With the below suggested renaming this is `file_age_win_file`. Same applies for `warning` and `critical`.

Reply:
I will update the documentation as well.